src/test/java/org/springframework/samples/petclinic/TestDBMethods.java

```diff
@@ -0,0 +1,310 @@
+import org.springframework.samples.petclinic.vet.Vet;
+import org.springframework.samples.petclinic.vet.VetRepository;
+import org.springframework.samples.petclinic.visit.Visit;
+import org.springframework.samples.petclinic.visit.VisitRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@Testcontainers
+public class TestDBMethods {
+	@Autowired
+	private VetRepository vets;
+	@Autowired
+	private OwnerRepository owners;
+	@Autowired
+	PetTypeRepository petTypes;
+	@Autowired
+	PetRepository pets;
+	@Autowired
+	VisitRepository visits;
+
+	@Container
+	private static final TarantoolCartridgeContainer container =
+		// Pass the classpath-relative paths of the instances configuration and topology script files
+		new TarantoolCartridgeContainer("cartridge/instances.yml", "cartridge/topology.lua")
+			// Point out the classpath-relative directory where the application files reside
+			.withDirectoryBinding("cartridge")
+			.withRouterHost("localhost") // Optional, "localhost" is default
+			.withRouterPort(3301) // Binary port, optional, 3301 is default
+			.withAPIPort(8081) // Cartridge HTTP API port, optional, 8081 is default
+			.withRouterUsername("admin") // Specify the actual username, default is "admin"
+			.withRouterPassword("secret-cluster-cookie")
+			.withUseFixedPorts(true)
+			.withReuse(true); // allows to reuse the container build once for faster testing
+
+
+	@BeforeAll
+	public static void init_cluster() throws Exception {
+		startCluster();
+	}
+
+	protected static void startCluster() throws Exception {
+		if (!container.isRunning()) {
+			container.start();
+			container.executeCommand("return require('migrator').up()").get();
+			container.executeCommand("require('data')").get();
+		}
+	}
+
+	protected String repeatString(String string, int amount) {
+		String result = "";
+		for (int i = 0; i < amount; ++i) {
+			result += string;
+		}
+
+		return result;
+	}
+
+	protected UUID genSimpleUuid(String n_string) {
+		return UUID.fromString(repeatString(n_string, 8) + "-" + repeatString(n_string, 4) +
+			"-" + repeatString(n_string, 4) + "-" + repeatString(n_string, 4) +
+			"-" + repeatString(n_string, 12));
+	}
+
+	protected UUID genSimpleUuid(int n) {
+		String n_string = String.valueOf(n);
+		return genSimpleUuid(n_string);
+	}
+
+	protected LocalDate fromStringToTimestamp(String string) {
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+		return LocalDate.parse(string, formatter);
+	}
+
+	protected void compairSpecialties(Specialty first_specialty, Specialty second_specialty) {
+		try {
+			assertThat(first_specialty.getId()).isEqualTo(second_specialty.getId());
+			assertThat(first_specialty.getName()).isEqualTo(second_specialty.getName());
+		} catch (NullPointerException e) {
```

Comment:
NullPointerException  does not require try/catch block if you want to do some action in case of NullPointerException please add this action, for example logging

Reply:
What should I do to ignore NullPointerException and continue testing?